README.adoc

```diff
@@ -54,38 +67,29 @@ If you use Nix, these tools are provided for you via `shell.nix`, and you do *no
 * If you want to build our Haskell packages with https://haskellstack.org/[`stack`], then install it.
 * If you want to build our Agda code, then install https://github.com/agda/agda[Agda] and the https://github.com/agda/agda-stdlib[standard library].
 
-=== How to get started using the platform
-
-The https://github.com/input-output-hk/plutus-starter[`plutus-starter`] repository contains a starter setup.
-
 === How to build the Haskell packages and other artifacts
 
 [[building-with-nix]]
 ==== How to build Haskell packages and other artifacts with Nix
 
-Run `nix build -f default.nix plutus.haskell.packages.plutus-core.components.library`
-from the root to build the Plutus Core library.
+Run `nix build -f default.nix plutus.haskell.packages.plutus-core.components.library` from the root to build the Plutus Core library.
 
-See <<nix-build-attributes>> to find out
-what other attributes you can build.
+See <<nix-build-attributes>> to find out what other attributes you can build.
 
 ==== How to build Haskell packages with `cabal`
```

Comment:
Should we mention here that you can use cabal _within_ nix? I assume this "build with cabal" here means build with cabal _outside_ of nix; because for example, the note about "R" doesn't make sense; as R is already present in the nix environment ... ?

Reply:
Yes, in principle this was supposed to indicate that you could do it with cabal _outside_ Nix. But I'm not really sure who the intended audience of this is any more: for people wanting to work on it, there is far more detail in `CONTRIBUTING` which  covers that stuff, and I can't imagine there are really any people who want to just build the libraries... for fun? Hmm.